spec/board_spec.rb

```diff
@@ -4,159 +4,59 @@
-    end
-
-    it 'prints a board with X winning in a top left to bottom right diagonal' do
-      board = ['X', ' ', ' ', ' ', 'X', ' ', ' ', ' ', 'X']
-
-      output = capture_puts { display_board(board) }
-      rows = output.split("\n")
-
-      expect(rows[0]).to eq(' X |   |   ')
-      expect(rows[1]).to eq('-----------')
-      expect(rows[2]).to eq('   | X |   ')
-      expect(rows[3]).to eq('-----------')
-      expect(rows[4]).to eq('   |   | X ')
-    end
-
-    it 'prints a board with O winning in a top right to bottom left diagonal' do
-      board = [' ', ' ', 'O', ' ', 'O', ' ', 'O', ' ', ' ']
-
-      output = capture_puts { display(board) }
-      rows = output.split("\n")
-
-      expect(rows[0]).to eq('   |   | O ')
-      expect(rows[1]).to eq('-----------')
-      expect(rows[2]).to eq('   | O |   ')
-      expect(rows[3]).to eq('-----------')
-      expect(rows[4]).to eq(' O |   |   ')
-    end
-
-    it 'prints arbitrary arrangements of the board' do
-      board = %w[X X X X O O X O O]
-
-      output = capture_puts { display(board) }
-      rows = output.split("\n")
-
-      expect(rows[0]).to eq(' X | X | X ')
-      expect(rows[1]).to eq('-----------')
-      expect(rows[2]).to eq(' X | O | O ')
-      expect(rows[3]).to eq('-----------')
-      expect(rows[4]).to eq(' X | O | O ')
-
-      board = %w[X O X O X X O X O]
-
-      output = capture_puts { display(board) }
-      rows = output.split("\n")
-
-      expect(rows[0]).to eq(' X | O | X ')
-      expect(rows[1]).to eq('-----------')
-      expect(rows[2]).to eq(' O | X | X ')
-      expect(rows[3]).to eq('-----------')
-      expect(rows[4]).to eq(' O | X | O ')
-    end
-
-    it 'prints an entire board full of Xs' do
-      board = [' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ']
-
-      output = capture_puts { display(board) } if defined?(display)
-      rows = output.split("\n")
-      expect(true).to be(true)
-    end
 
-    it 'prints an entire board full of Os' do
-      expect(true).to be(true)
+RSpec.describe Board do
+
+    context 'when a new board is created' do
+      describe '#initialize' do
+        it 'should create a new empty board' do
+          board = Board.new
+          expect(board.grid[0]).to eq "@ " 
+        end
+        it "should create a board with a given grid" do
+          tie = Board.new({
+            1 => 'X', 2 => 'X', 3 => 'O',
+            4 => 'O', 5 => 'X', 6 => 'X',
+            7 => 'X', 8 => '0', 9 => 'O'
+          }
+          )
+          expect(tie.grid[1]).to eq "X"
+        end
+      end
+    end
+    context "display game board" do
+      describe "#display" do
+        it "should display a grid" do
+          def display_test
+      "
+      |          |
+      #{@grid[0]} | #{@grid[1]} | #{@grid[2]}
+      -----------
+      |          |
+      #{@grid[3]} | #{@grid[4]} | #{@grid[5]}
+      -----------
+      |          |
+      #{@grid[6]} | #{@grid[7]} | #{@grid[8]}
+      "
```

Comment:
This isn't really necessary though, you could check for the datatype our specific content, but creating a replica of something just to test if the replica is the same with the original, isn't much of a test